src/components/itemHelper.js

```diff
@@ -331,6 +331,19 @@ export function supportsMediaSourceSelection (item) {
     return true;
 }
 
+export function sortTracks (trackA, trackB) {
+    if (Number(trackA.IsExternal) < Number(trackB.IsExternal)) return -1;
+    if (Number(trackB.IsExternal) < Number(trackA.IsExternal)) return 1;
+
+    if (Number(trackA.IsForced) < Number(trackB.IsForced)) return -1;
+    if (Number(trackB.IsForced) < Number(trackA.IsForced)) return 1;
+
+    if (Number(trackB.IsDefault) < Number(trackA.IsDefault)) return -1;
+    if (Number(trackB.IsDefault) < Number(trackA.IsDefault)) return 1;
```

Comment:
```suggestion
    if (Number(trackA.IsDefault) < Number(trackB.IsDefault)) return 1;
```
I assumed you need `Default` to go first.